### ISSUE TITLE
Getting started with side projects

### ISSUE THREAD

**Issue by varughese** [PittCSWiki/pittcswiki#231]
skills/projects

Michael wehar (past PTI at pitt and gave talks to the club a few times sent us an email also linking this - 
http://michaelwehar.com/documents/SideProjects.pdf which has good info and said we could link

**Comment by MichaelWehar** [comment]
Thanks so much for adding this as an issue!

I really enjoy working on coding projects with students and gave this talk for Pitt CSC.  Please feel welcome to share it.  :)

Also, another PTI named Muneeb Alvi wrote a CS job search guide: https://onedrive.live.com/view.aspx?resid=2203166754AF485B!486&ithint=file%2cpptx&authkey=!AOxaiO5tEkV6RkY

PittCSWiki is really cool!  Keep up the great work.  :)

**Comment by Richie78321** [comment]
We don’t have a solid guide for this yet. I’m thinking we should keep this open for now

**Comment by Zmwang622** [comment]
Fair enough, I thought the new Getting Started with X guides would replace this one but makes sense that we should write this one up alongside the others. 

Apologies for the closing the issue!
